swedish-oidc-profile.md

```diff
@@ -349,50 +347,56 @@ Relying Parties MUST follow the requirements in section [3.1.3.7] of \[[OpenID.C
-interoperability, a claim that represents an identity value that already has a definition in \[[OIDC.Sweden.Attr](#attr-spec)\]
-MUST NOT be re-defined with a new claim name in profiles extending the Swedish OpenID Connect profile.
-
-> As an example: An OpenID Provider compliant with this profile MUST not invent its own claim definition of a Swedish
-personal identity number. It MUST use the `https://id.oidc.se/claim/personalIdentityNumber` defined in 
-\[[OIDC.Sweden.Attr](#attr-spec)\].
-
 <a name="userinfo-endpoint"></a>
 ### 4.1. UserInfo Endpoint
 
-An OpenID Provider compliant with this profile SHOULD support releasing claims from the UserInfo endpoint. If supported the OpenID Provider MUST follow the requirements from section 5.3 of \[[OpenID.Core](#openid-core)\].
+An OpenID Provider compliant with this profile MUST support releasing claims from the UserInfo endpoint
+and MUST follow the requirements from section 5.3 of \[[OpenID.Core](#openid-core)\].
+
+Access to the UserInfo endpoint MUST be denied if a valid access token is not presented.
 
 Responses from the UserInfo endpoint MUST be signed.
 
-The OpenID Provider MUST NOT release any user identity claims other than the mandatory `sub` claim if they are not explicitly requested in the original authentication request via the `claims` parameter, see [2.1.6](#the-claims-parameter), or by a standard scope (see below).
+See section [4.2](#claims-release-requirements) below for claims release requirements via the UserInfo
+endpoint.
+
+Relying Parties MUST follow section 5.3.4 of \[[OpenID.Core](#openid-core)\] when validating a
+UserInfo response message.
 
 <a name="claims-release-requirements"></a>
 ### 4.2. Claims Release Requirements
 
 OpenID Providers MUST return claims on a best effort basis. However, an OpenID Provider asserting it can provide a user claim does not imply that this data is available for all its users. Relying Parties MUST be prepared to receive partial data. 
 
-An OpenID Provider compliant with this profile MUST NOT release any identity claims<sup>1</sup> in the ID token, or via the UserInfo endpoint, if they have not been explicitly requested via the `scope` and/or `claims` request parameters, or by a policy known, and accepted, by the involved parties. Furthermore, an OpenID Provider MUST NOT release any claims to a Relying Party that has not been authorized to receive
-them. 
+An OpenID Provider compliant with this profile MUST NOT release any identity claims in the ID token, or via the UserInfo endpoint, if they have not been explicitly requested via `scope` and/or `claims` request parameters, or by a policy known, and accepted, by the involved parties. 
 
-Section 5.4 of \[[OpenID.Core](#openid-core)\] states:
+The above requirement does not include the mandatory `sub` claim, and claims that do not reveal identity
+information about the user, for example, transaction identifiers or claims holding information about
+the authentication process.
 
```

Comment:
There should be a text explaining that the rules below only applies to identity claims